tests/integration/WatchmanTestCase.py

```diff
@@ -29,6 +29,13 @@
 def norm_path(name):
     return os.path.normcase(os.path.normpath(name))
 
+def conv_path_to_bytes(path):
+    if (not pywatchman.compat.PYTHON3 and isinstance(path, unicode)) or \
```

Comment:
Could you add a TODO comment here saying that this will go away/no longer need to be normalized once https://github.com/facebook/watchman/wiki/Better-Unicode-handling-plan is done?